packages/kuma-gui/src/app/control-planes/locales/en-us/index.yaml

```diff
@@ -18,3 +18,7 @@ main-overview:
       title: 'Zones'
     meshes:
       title: 'Meshes'
+  action_menu:
+    toggle_button: 'Actions'
+    items:
+      hostname_generators: HostnameGenerators
```

Comment:
Is the label fine?

Reply:
If we are staying consistent, I think this needs a space at least for now. We use `Mesh Services`/`Mesh External Services` etc elsewhere. TBH I kinda like using the actual config name (no space), but if we want to change that we should do it everywhere in one go.